terraform-plans/templates/github/snap_tics.yaml.tftpl

```diff
@@ -23,25 +23,26 @@ jobs:
         run: |
           sudo apt update
           python -m pip install --upgrade pip
-          # tics action requires flake8 and pylint to be installed on the runner
-          # https://github.com/tiobe/tics-github-action/issues/410
-          python -m pip install flake8 pylint
           # pin tox to the current major version to avoid
           # workflows breaking all at once when a new major version is released.
           python -m pip install 'tox<5'
 
-      # The snap is built already in the check workflow and available as an artifact,
-      # but we want this TICS workflow to be independent,
-      # so we can run the workflow periodically or on demand.
+        # The snap is built already in the check workflow and available as an artifact,
+        # but we want this TICS workflow to be independent,
+        # so we can run the workflow periodically or on demand.
       - name: Build the snap
         id: build
         uses: canonical/action-build@v1
 
       - name: Test with tox and produce coverage report
-        run: tox -e tics
+        run: tox --workdir /tmp/tox -e tics
 
       - name: Run TICS analysis
         uses: tiobe/tics-github-action@v3
+        env:
+          # Add the the tox bin directory to the PATH so that the TICS action can find the tests dependencies.
+          # Other paths are added to the PATH to make sure that the TICS action can find the required binaries.
+          PATH: "/tmp/tox/tics/bin:/snap/bin:/home/runner/.local/bin:/home/runner/.cargo/bin:/bin:/usr/bin:/usr/local/bin:"
```

Comment:
```suggestion
          PATH: "/tmp/tox/tics/bin:$PATH"
```

Does this work in github workflows? That would save you needing to hardcode the rest of the path

Reply:
I checked, didn't work, it was missing curl